notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>I think the 100_datamodules link is broken</p><p>Also, are you sure you want to link the 701a_aupimo to the raw notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
fixed both, also in previous notebooks